accepted/2020/workloads/workload-manifest.md

```diff
@@ -82,7 +82,9 @@ Library packs are normal NuGet packages. When a library pack is installed, the n
 
 This location is used as a local feed by NuGet, not a fallback folder. If ones of these packages gets used, it will be extracted into the global packages folder. If the SDK is updated and the library pack is removed or replaced with a newer version, projects that have already been created and restored will continue to be able to use the extracted copy from the global packages folder.
 
-This is intended to be used to pre-download NuGet packages referenced by templates so that a workload can work offline after installation. It should be used sparingly i.e. only for NuGets referenced by the core templates with default or popular options.
+Library packs are intended to be used to pre-download NuGet packages referenced by templates so that a workload can work offline after installation. Core workloads should include library packs very sparingly, if at all. Instead, it is strongly recommended that library packs are contained in separate workloads with the ID suffix `-offline-templates`. An offline template workload should only contain library packs referenced by templates, and should extend the workload that contains the templates.
+
+> NOTE: SDK targets should **not** implicitly add `PackageReference` items. They should instead add `FrameworkReference` items that resolve to runtime packs and targeting packs.
```

Comment:
I wouldn't phrase this as strongly, as I think .NET MAUI is going to have implicit PackageReference items.

@jonathanpeppers

Reply:
Right now the plan with dotnet/maui is to have some implicit `PackageReference` for things like:
* https://www.nuget.org/packages/Microsoft.ProjectReunion/
* https://www.nuget.org/packages/Xamarin.Google.Android.Material/
* https://www.nuget.org/packages/Xamarin.AndroidX.AppCompat/
* https://www.nuget.org/packages/Microsoft.Maui.Graphics/ - supposed to be a standalone NuGet

I agree with `workloads should include library packs very sparingly`, but it doesn't seem like dotnet/maui is doing that at the moment. dotnet/maui probably wouldn't be a good fit with the `-offline-templates` naming for these either, because they are dependencies for any app.

/cc @Redth